Dockerfile

```diff
@@ -5,6 +5,7 @@ WORKDIR /app
 # copy repo contents
 COPY setup.py README.md ./
 COPY optimade ./optimade
+COPY providers/providers.json ./optimade/server/data/
```

Comment:
Should we also run `git submodule init` and `git submodule update` or something?

Reply:
This isn't possible in the Docker, since we purposely copy the minimum amount of data into the Dockerfile, which excludes all the git info. The expectation is what whomever / whatever is building the docker is responsible for making a full checkout of the repo, including git submodules.